.github/workflows/release.yml

```
@@ -0,0 +1,94 @@
+name: Publish Package on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup
+
+      - name: Build contracts
+        run: turbo build
```

Improve package scoping
```suggestion
        run: turbo build --filter=!'docs'
```